core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -81,11 +158,7 @@ function ampContent() {
         // errors in video, because video will be stripped out.
         // @TODO: remove this, when Amperize support video transform
         $('video').children('source').remove();
-
-        // Vimeo iframe e. g. come with prohibited attributes
-        // @TODO: remove this, when Amperize supports HTML sanitizing
-        $('amp-iframe').removeAttr('webkitallowfullscreen');
-        $('amp-iframe').removeAttr('mozallowfullscreen');
+        $('video').children('track').remove();
```

Comment:
you can do as a separate PR, this is already green. so would like to merge now 👍